packages/react-sdk/package.json

```diff
@@ -46,8 +46,8 @@
     "@rollup/plugin-typescript": "^11.1.6",
     "@stream-io/audio-filters-web": "workspace:^",
     "@stream-io/video-styling": "workspace:^",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
```

Comment:
I just noticed this but I think we need to list `@types/react` and `@types/react-dom` in the dev dependencies. It works now because we have them hoisted to the root.

Reply:
Also, I'm not sure if we need `react-dom` as well, as we aren't using anything from it in this package.
What do you think?

Reply:
I agree with both points. Fixed in https://github.com/GetStream/stream-video-js/pull/1365/commits/42f550f224dd20d545dc4ec7fca30e8c7a6efc48